job/common/src/main/java/alluxio/job/wire/JobServiceSummary.java

```diff
@@ -40,12 +40,12 @@
 
   /**
    * Constructs a new instance of {@link JobServiceSummary} from a
-   * collection of {@link JobInfo} it possesses.
+   * collection of {@link PlanInfo} it possesses.
    *
-   * @param jobInfos Collection of {@link JobInfo}
+   * @param jobInfos Collection of {@link PlanInfo}
```

Comment:
Would prefer to keep this as PlanInfos especially because the constructor of this should be internal use.